ui/src/app/components/extend-initial-credits-modal.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import { Profile } from 'generated/fetch';
+
+import { Button, StyledExternalLink } from 'app/components/buttons';
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalTitle,
+} from 'app/components/modals';
+import { Spinner } from 'app/components/spinners';
+import { profileApi } from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import { displayDateWithoutHours, minusDays } from 'app/utils/dates';
+import { notificationStore, serverConfigStore } from 'app/utils/stores';
+import { supportUrls } from 'app/utils/zendesk';
+
+import { AoU } from './text-wrappers';
+
+interface Props {
```

Comment:
I prefer to group these with the component which uses them, to make it clearer which one does.  so here I'd move this below ExtensionDescription